files/site.tfvars.in

```diff
@@ -24,7 +24,7 @@ dns_forwarders              = "192.168.88.1;$DNS_BACKUP_SERVER"
 chrony_config               = true
 chrony_config_servers       = [$CHRONY_CONFIG_SERVERS]
 
-install_playbook_tag        = "9a5128fc58ed86aa332128314181dc9ab86334c6"
+install_playbook_tag        = "master"
```

Comment:
I would feel better setting this to a specific commit if something is missing or broken.   If a bug is introduced here, we may lose a day or two as we are tracking other problems.   We are about to have ocs-ci running in automated fashion every day on a couple of servers.   That is going to take extra time and resources to keep track of.

Reply:
Okay, I've pointed it to a specific commit now.